Build/Scripts/seajax_v2.py

```diff
@@ -93,9 +94,11 @@ def build_specific(target, type):
             "npx",
             "babel",
             PATH_COMPILED_FILES,
-            "--quiet",
             "--config-file",
             "../../.babelrc.json",
+            "--extensions",
+            ".jsx",
```

Comment:
Compiling all JS files caused problems (infinite recursion on Babel generated `_typeof` helper, maybe because it’s duplicated), so I compromised and only opted-in JSX files for Babel compilation.